Automattic/vip-go-mu-plugins
# Add/jetpack 10.5 beta

## Description
Added subtree for Jetpack 10.5-beta
Added logic for WP 5.8 compatibility with Jetpack 10.5+

## Changelog Description
Jetpack 10.5-beta has been added to MU-plugins. [Users can pin at Jetpack 10.5](https://docs.wpvip.com/how-tos/test-jetpack-on-vip-go/#h-pinning-to-a-version).

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Check out PR.
1. [Pin Jetpack to 10.5 ](https://docs.wpvip.com/how-tos/test-jetpack-on-vip-go/#h-pinning-to-a-version)
1. Go to `wp-admin` > `Jetpack` 
1. Verify Jetpack version is 10.5-beta
1. Verify cookies are delicious.
